### ISSUE TITLE
Command and Conquer: Red Alert 3 Demo (24710) (the full game probably suffers from the same issue)

### ISSUE THREAD

**Issue by Steap** [ValveSoftware/Proton#2029]
# Compatibility Report
- Name of the game with compatibility issues: Command and Conquer: Red Alert 3 Demo
- Steam AppID of the game: 24710

## System Information
- GPU: Intel HD Graphics 4000
- Driver/LLVM version: 3.0 Mesa 18.1.9
- Kernel version: 4.18.0-2-amd64
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/Steap/b4cc3f1bfef34b19b2299f295fdd6a4a
- Proton version: 3.7-8

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
When running the first misson of the tutorial ("Intro to the basics"), the screen is mostly... purple. See the attached screenshot. 
![red_alert_3_bug](https://user-images.githubusercontent.com/416834/48974917-dd4d6480-f065-11e8-8a57-0f2742f42be8.png)

This happens when the graphics presets (Settings -> Graphics) are set to "low" or "lowest". With "medium", the game looks perfectly normal.

## Reproduction
Run the first mission of the tutorial with "low" or "lowest" graphics presets.

**Comment by nded03** [comment]
Seems okay on my end.
![20181128194902_1](https://user-images.githubusercontent.com/44517119/49198185-d0bb5b80-f346-11e8-9d3c-fa74fc15118b.jpg)

**Comment by Steap** [comment]
@nded03 Could you post your system info in a gist?

**Comment by nded03** [comment]
https://gist.github.com/nded03/e9b805fd4dfb64b16a68487b6974db09

**Comment by kisak-valve** [comment]
Reproduced locally on an Ivybridge, Kabylake, RX 480 (18.2.x), and a GTX 950 (396.54). Dropping the mesa label since the graphics vendors are affected equally.

@nded03, are you testing with the lowest graphics preset in-game?

**Comment by nded03** [comment]
@kisak-valve yes, but I just realized that I only have the Red Alert Uprising expansion rather than the original. I downloaded the demo and was able to reproduce the issue in it. Seems to be entirely dependent on texture quality; any setting other than high will result in pink terrain.
![20181130182715_1](https://user-images.githubusercontent.com/44517119/49323329-67feeb00-f4ce-11e8-92c6-b700d51addbd.jpg)

**Comment by Industrial** [comment]
> @kisak-valve yes, but I just realized that I only have the Red Alert Uprising expansion rather than the original. I downloaded the demo and was able to reproduce the issue in it. Seems to be entirely dependent on texture quality; any setting other than high will result in pink terrain.

You don't get this error with Red Alert Uprising? https://github.com/ValveSoftware/Proton/issues/6459
